Shopify/cli
# Add a script to create test themes

### WHY are these changes introduced?

It's currently hard to test all possible configurations of how the shopify tooling is installed for themes.

### WHAT is this pull request doing?

Adds a node script to create test themes using homebrew, local cli repo, and npm global nightly packages. 

### How to test your changes?

First of all unset the following env vars:
- `unset GEM_HOME`
- `unset GEM_PATH`

We need to do this otherwise the shadowenv variables are messing with where gems are looked up. I am investigating if we should always pass it when running `bundle` commands.

Then try once for each supported way:
- local
  - `node bin/create-test-theme.js -s <YOUR_STORE> --cleanup -i local`
  - Open http://127.0.0.1:9292/
  - Change the background color of the first div element in `sections/announcement-bar.liquid` to `color-inverse`
  - Check that the theme is hot reloaded
  - press `Ctrl+C` to continue
  - script should continue and terminate successfully
- homebrew
  - `node bin/create-test-theme.js -s <YOUR_STORE> --cleanup -i homebrew`
  - Open http://127.0.0.1:9292/
  - Change the background color of the first div element in `sections/announcement-bar.liquid` to `color-inverse`
  - Check that the theme is hot reloaded
  - press `Ctrl+C` to continue
  - script should continue and terminate successfully
- npm globally
  - `node bin/create-test-theme.js -s <YOUR_STORE> --cleanup -i npm-global`
  - Open http://127.0.0.1:9292/
  - Change the background color of the first div element in `sections/announcement-bar.liquid` to `color-inverse`
  - Check that the theme is hot reloaded
  - press `Ctrl+C` to continue
  - script should continue and terminate successfully

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
